### PR TITLE
Add German translation for documentation

### PR DESCRIPTION
**Pull request by kolja-lucht**
## Summary

Adding German translation for the documentation.

## Related Issues
#738 

## Checklist

- [x] This pull request has a meaningful title and description.
- [x] I have read and followed the [Contributing guidelines](https://github.com/danielschuster-muc/potter-db/blob/master/CONTRIBUTING.md).
- [x] My code follows the conventions and coding style of this project.
- [x] All tests pass and I have added tests for my changes 
- [x] I have updated necessary documentation

### CODE REVIEW

**Comment by kolja-lucht** [comment]
Hey @danielschuster-muc, my translations are ready for review 😊

**Comment by danielschuster-muc** [comment]
> Hey @danielschuster-muc, my translations are ready for review 😊

I'll have a look at those later this evening. Takes a bit longer to review than #793 😂.

**Comment by kolja-lucht** [comment]
Fair enough 😆

File: docs/pages/_meta.de.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "index": "Introduction",
```
Comment (danielschuster-muc) [inline docs/pages/_meta.de.json:2]:
```suggestion
  "index": "Einführung",
```

File: docs/pages/apis/rest.de.mdx [hunk 2]
```diff
@@ -0,0 +1,161 @@
+# REST API
+
+Unsere REST-API bietet eine unkomplizierte Möglichkeit, mit der Potter DB zu interagieren, wobei die JSON:API- und OAS-Spezifikation eingehalten wird. In diesem Abschnitt behandeln wir die wichtigsten Aspekte der Verwendung der REST-API, einschließlich der verfügbaren Endpunkte und der Formatierung deiner Anfragen und Antworten.
+
+## Verfügbare Endpunkte
+
+Um auf Daten aus der REST-API zuzugreifen, musst du eine "GET"-Anfrage an einen der folgenden Endpunkte schicken:
```
Comment (danielschuster-muc) [inline docs/pages/apis/rest.de.mdx:7]:
```suggestion
Um auf Daten aus der REST-API zuzugreifen, musst du eine `GET`-Anfrage an einen der folgenden Endpunkte schicken:
```
It's nicer to use a backtick (`). Please also change it where needed.

File: docs/pages/apis/rest.de.mdx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+**Zaubertränke**
+- `/v1/potions`: Abruf einer Liste von Zaubertränken.
+- `/v1/potions/{id}`: Abruf eines Zaubertranks anhand seiner ID.
+
+**Zaubersprüche**
+- `/v1/spells`: Abruf einer Liste von Zaubersprüchen.
+- `/v1/spells/{id}`: Abruf eines Zauberspruchs anhand seiner ID.
+
+IDs müssen als UUIDs oder Kennung (slugs) angegeben werden.
```
Comment (danielschuster-muc) [inline docs/pages/apis/rest.de.mdx:34]: Same here with the Kennung.

File: docs/pages/apis/rest.de.mdx [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+</details>
+
+## OpenAPI Spezifikation (OAS)
+
+Um die Integration zu erleichtern, stellen wir ein [OpenAPI Spezifikation](https://openapis.org) Dokument zur Verfügung, das die Endpunkte, Datenmodelle und Anfrage-/Antwortschemata der API beschreibt.
+Die API entspricht der Version 3.0.3 der OAS-Spezifikation.
+Du kannst unser aktuelles OAS document [hier](https://api.potterdb.com/v1/openapi.json) finden.
```
Comment (danielschuster-muc) [inline docs/pages/apis/rest.de.mdx:42]:
```suggestion
Unser aktuelles OAS Dokument ist [hier](https://api.potterdb.com/v1/openapi.json) zu finden.
```

File: docs/pages/apis/rest.de.mdx [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+
+## JSON:API Format
+
+Unsere REST-API entspricht der [JSON:API-Spezifikation](https://jsonapi.org/), die eine einheitliche Methode zur Strukturierung von API-Anfragen und -Antworten bietet. Hier ist ein Überblick über die wichtigsten Merkmale von JSON:API im Zusammenhang mit unserer API:
+
+- **Ressourcen Elemente**: Jede Ressource ist ein Objekt mit einem Typ (`type`), ID (`id`) und Attributen (`attributes`).
+- **Verbindungen**: Ressourcen können mit anderen Ressourcen verbunden sein, z. B. ist eine `Buch`-Ressource mit einer `Kapitel`-Ressource verbunden und umgekehrt.
```
Comment (danielschuster-muc) [inline docs/pages/apis/rest.de.mdx:49]:
```suggestion
- **Relationen**: Ressourcen können mit anderen Ressourcen verbunden sein, z. B. ist eine `Buch`-Ressource mit einer `Kapitel`-Ressource verbunden und umgekehrt.
```

File: docs/pages/_meta.de.json [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  "getting-started": "Zum Einstieg",
```
Comment (danielschuster-muc) [inline docs/pages/_meta.de.json:3]:
```suggestion
  "getting-started": "Erste Schritte",
```
sounds better

File: docs/pages/apis/graphql.de.mdx [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+        }
+      }
+    }
+  }
+  character(slug: "harry-potter") {
+    name
+  }
+}
+```
+In dieser Abfrage werden der Titel (`title`) und das Erscheinungsdatum (`releaseDate`) der ersten drei Bücher, sowie der Titel (`title`) und die Kennung (`lug`) aller Kapitel jedes Buches abgefragt. Außerdem wird der Name (`name`) der Figur mit der Kennung (`slug`) "harry-potter" abgefragt.
```
Comment (danielschuster-muc) [inline docs/pages/apis/graphql.de.mdx:43]: I would prefer to keep it as slug, "Kennung" does not really fit the word imo. Please change it everywhere.

File: docs/pages/resources/books.de.mdx [hunk 8]
```diff
@@ -0,0 +1,81 @@
+# Bücher
+Diese Ressource bietet Informationen über Bücher aus dem Harry Potter-Universum.
```
Comment (danielschuster-muc) [inline docs/pages/resources/books.de.mdx:2]:
```suggestion
Diese Ressource bietet Informationen über Bücher aus dem Harry-Potter-Universum.
```
Please also change it in other resources, too.

File: docs/pages/resources/books.de.mdx [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+      },
+      {
+        name: "summary",
+        type: "Zeichenfolge",
+        description: "Die Buchzusammenfassung.",
+        filterable: true,
+      },
+      {
+        name: "slug",
+        type: "Zeichenfolge",
```
Comment (danielschuster-muc) [inline docs/pages/resources/books.de.mdx:55]: same here with the slug

File: docs/pages/resources/characters.de.mdx [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+      { name: "died", type: "Zeichenfolge", description: "Datum und Ort des Todes der Figur.", filterable: true },
+      { name: "eye_color", type: "Zeichenfolge", description: "Augenfarbe der Figur.", filterable: true },
+      { name: "family_member", type: "array", description: "Liste der Familienmitglieder dieser Figur.", filterable: true },
+      { name: "gender", type: "Zeichenfolge", description: "Das Geschlecht dieser Figur.", filterable: true },
+      { name: "hair_color", type: "Zeichenfolge", description: "Haarfarbe der Figur.", filterable: true },
+      { name: "height", type: "Zeichenfolge", description: "Die Größe der Figur in Zoll.", filterable: true },
+      { name: "house", type: "Zeichenfolge", description: "Das Hogwarts Haus der Figur.", filterable: true },
+      { name: "image", type: "Zeichenfolge", description: "Link zu einem Bild dieser Figur.", filterable: false },
+      { name: "jobs", type: "Array", description: "Liste von Tätigkeiten, die die Figur ausgeführt hat.", filterable: true },
+      { name: "name", type: "Zeichenfolge", description: "Titel der Figur.", filterable: true },
```
Comment (danielschuster-muc) [inline docs/pages/resources/characters.de.mdx:31]:
```suggestion
      { name: "name", type: "Zeichenfolge", description: "Name der Figur.", filterable: true },
```

File: docs/pages/resources/characters.de.mdx [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+      { name: "nationality", type: "Zeichenfolge", description: "Nationalität der Figur.", filterable: true },
+      { name: "patronus", type: "Zeichenfolge", description: "Der Patronus der Figur.", filterable: true },
+      { name: "romances", type: "array", description: "Liste von Romanzen der Figur.", filterable: true },
+      { name: "skin_color", type: "Zeichenfolge", description: "Hautfarbe der Figur.", filterable: true },
+      { name: "slug", type: "Zeichenfolge", description: "Kennung um die Figur zu identifizieren.", filterable: false },
+      { name: "species", type: "Zeichenfolge", description: "Spezies der Figur.", filterable: true },
+      { name: "titles", type: "array", description: "Liste mit Namen, die die Position oder Job der Figur beschreiben.", filterable: true },
```
Comment (danielschuster-muc) [inline docs/pages/resources/characters.de.mdx:38]:
```suggestion
      { name: "titles", type: "array", description: "Liste mit Namen, die die Position oder den Job der Figur beschreiben.", filterable: true },
```

File: docs/pages/resources/characters.de.mdx [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+      <th>Name</th>
+      <th>Typ</th>
+      <th>Beschreibung</th>
+      <th>Filterbar</th>
+    </tr>
+  </thead>
+  <tbody>
+    {[
+      { name: "alias_names", type: "Array", description: "Eine Liste von Namen, unter denen diese Figur sonst bekannt war.", filterable: true },
+      { name: "animagus", type: "Zeichenfolge", description: "Der Animagus der Figur.", filterable: true },
```
Comment (danielschuster-muc) [inline docs/pages/resources/characters.de.mdx:18]:
```suggestion
      { name: "animagus", type: "Zeichenfolge", description: "Animagus der Figur.", filterable: true },
```

File: docs/pages/resources/characters.de.mdx [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+      { name: "blood_status", type: "Zeichenfolge", description: "Blut-Status der Figur.", filterable: true },
+      { name: "boggart", type: "Zeichenfolge", description: "Irrwicht der Figur.", filterable: true },
+      { name: "born", type: "Zeichenfolge", description: "Geburtsdatum und Geburtsort der Figur.", filterable: true },
+      { name: "died", type: "Zeichenfolge", description: "Datum und Ort des Todes der Figur.", filterable: true },
+      { name: "eye_color", type: "Zeichenfolge", description: "Augenfarbe der Figur.", filterable: true },
+      { name: "family_member", type: "array", description: "Liste der Familienmitglieder dieser Figur.", filterable: true },
+      { name: "gender", type: "Zeichenfolge", description: "Das Geschlecht dieser Figur.", filterable: true },
```
Comment (danielschuster-muc) [inline docs/pages/resources/characters.de.mdx:25]:
```suggestion
      { name: "gender", type: "Zeichenfolge", description: "Geschlecht dieser Figur.", filterable: true },
```

File: docs/pages/resources/characters.de.mdx [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+      { name: "hair_color", type: "Zeichenfolge", description: "Haarfarbe der Figur.", filterable: true },
+      { name: "height", type: "Zeichenfolge", description: "Die Größe der Figur in Zoll.", filterable: true },
```
Comment (danielschuster-muc) [inline docs/pages/resources/characters.de.mdx:27]:
```suggestion
      { name: "height", type: "Zeichenfolge", description: "Größe der Figur in Zoll.", filterable: true },
```

File: docs/pages/resources/characters.de.mdx (hunk 10 above)
Comment (danielschuster-muc) [inline docs/pages/resources/characters.de.mdx:31]:
```suggestion
      { name: "name", type: "Zeichenfolge", description: "Name der Figur.", filterable: true },
```

File: docs/pages/resources/characters.de.mdx [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+      { name: "house", type: "Zeichenfolge", description: "Das Hogwarts Haus der Figur.", filterable: true },
```
Comment (danielschuster-muc) [inline docs/pages/resources/characters.de.mdx:28]:
```suggestion
      { name: "house", type: "Zeichenfolge", description: "Hogwarts Haus der Figur.", filterable: true },
```

File: docs/pages/resources/books.de.mdx [hunk 16]
```diff
@@ -0,0 +1,81 @@
...
+      {
+        name: "author",
+        type: "Zeichenfolge",
+        description: "Autor des Buches.",
+        filterable: true,
+      },
+      {
+        name: "cover",
+        type: "Zeichenfolge",
+        description: "Die URL des Buch-Covers.",
```
Comment (danielschuster-muc) [inline docs/pages/resources/books.de.mdx:26]:
```suggestion
        description: "URL des Buch-Covers.",
```

File: docs/pages/resources/books.de.mdx [hunk 17]
```diff
@@ -0,0 +1,81 @@
...
+        filterable: false,
+      },
+      {
+        name: "dedication",
+        type: "Zeichenfolge",
+        description: "Die Widmung des Buchs.",
```
Comment (danielschuster-muc) [inline docs/pages/resources/books.de.mdx:32]:
```suggestion
        description: "Widmung des Buchs.",
```

File: docs/pages/resources/books.de.mdx [hunk 18]
```diff
@@ -0,0 +1,81 @@
...
+        filterable: true,
+      },
+      {
+        name: "pages",
+        type: "Integer",
+        description: "Die Seitenanzahl des Buchs.",
```
Comment (danielschuster-muc) [inline docs/pages/resources/books.de.mdx:38]:
```suggestion
        description: "Seitenanzahl des Buchs.",
```

File: docs/pages/resources/books.de.mdx [hunk 19]
```diff
@@ -0,0 +1,81 @@
...
+      {
+        name: "release_date",
+        type: "Datum",
+        description: "Erscheinungsdatum des Buchs.",
+        filterable: true,
+      },
+      {
+        name: "summary",
+        type: "Zeichenfolge",
+        description: "Die Buchzusammenfassung.",
```
Comment (danielschuster-muc) [inline docs/pages/resources/books.de.mdx:50]:
```suggestion
        description: "Buchzusammenfassung.",
```

File: docs/pages/resources/chapters.de.mdx [hunk 20]
```diff
@@ -0,0 +1,51 @@
...
+      <th>Beschreibung</th>
+      <th>Filterbar</th>
+    </tr>
+  </thead>
+  <tbody>
+    {[
+      {
+        name: "order",
+        type: "Integer",
+        description: "Die chronologische Reihenfolge des Kapitels im Buch.",
```
Comment (danielschuster-muc) [inline docs/pages/resources/chapters.de.mdx:20]:
```suggestion
        description: "Chronologische Reihenfolge des Kapitels im Buch.",
```

File: docs/pages/resources/potions.de.mdx [hunk 21]
```diff
@@ -0,0 +1,99 @@
...
+      {
+        name: "ingredients",
+        type: "Zeichenfolge",
+        description: "Zutaten, um diesen Zaubertrank zu brauen.",
+        filterable: true,
+      },
+      {
+        name: "manufacturers",
+        type: "Zeichenfolge",
+        description: "Die Hersteller dieses Zaubertranks.",
```
Comment (danielschuster-muc) [inline docs/pages/resources/potions.de.mdx:56]:
```suggestion
        description: "Hersteller dieses Zaubertranks.",
```

File: docs/pages/resources/spells.de.mdx [hunk 22]
```diff
@@ -0,0 +1,87 @@
...
+      {
+        name: "image",
+        type: "Zeichenfolge",
+        description: "Link zu einem Bild des Zauberspruchs.",
+        filterable: false,
+      },
+      {
+        name: "incantation",
+        type: "Zeichenfolge",
+        description: "Die Beschwörungsformel für diesen Zauberspruch.",
```
Comment (danielschuster-muc) [inline docs/pages/resources/spells.de.mdx:50]:
```suggestion
        description: "Beschwörungsformel für diesen Zauberspruch.",
```

File: docs/pages/resources/spells.de.mdx [hunk 23]
```diff
@@ -0,0 +1,87 @@
...
+        filterable: true,
+      },
+      {
+        name: "light",
+        type: "Zeichenfolge",
+        description: "Das Licht des Zauberspruchs.",
```
Comment (danielschuster-muc) [inline docs/pages/resources/spells.de.mdx:56]:
```suggestion
        description: "Licht des Zauberspruchs.",
```

File: docs/pages/resources/spells.de.mdx [hunk 24]
```diff
@@ -0,0 +1,87 @@
...
+        filterable: true,
+      },
+      {
+        name: "name",
+        type: "Zeichenfolge",
+        description: "Der Name des Zauberspruchs.",
```
Comment (danielschuster-muc) [inline docs/pages/resources/spells.de.mdx:62]:
```suggestion
        description: "Name des Zauberspruchs.",
```

File: docs/theme.config.tsx [hunk 25]
```diff
@@ -60,6 +64,9 @@ const config: DocsThemeConfig = {
       case "sk":
         text = "Naposledy upravené";
         break;
+      case "de":
+        text = "Letztes Update am";
```
Comment (danielschuster-muc) [inline docs/theme.config.tsx:68]:
```suggestion
        text = "Letzte Änderung am";
```

File: docs/theme.config.tsx [hunk 26]
```diff
@@ -23,6 +23,8 @@ const config: DocsThemeConfig = {
           return "Editar esta página en GitHub →";
         case "sk":
           return "Upraviť stránku cez GitHub →";
+        case "de":
+          return "Editiere diese Seite auf GitHub →";
```
Comment (danielschuster-muc) [inline docs/theme.config.tsx:27]:
```suggestion
          return "Bearbeite diese Seite auf GitHub →";
```

File: docs/pages/apis/graphql.de.mdx [hunk 27]
```diff
@@ -0,0 +1,46 @@
+import { Callout } from "nextra-theme-docs";
+
+<Callout type="warning" emoji="⚠️">
+  Die GraphQL-API ist noch in Arbeit und einige Funktionen wie Filtern und Sortieren sind derzeit noch **nicht** implementiert! Änderungen an der API sind zu erwarten.
+</Callout>
+
+# GraphQL API
```
Comment (danielschuster-muc) [inline docs/pages/apis/graphql.de.mdx:7]:
```suggestion
# GraphQL API

```

File: docs/pages/apis/rest.de.mdx [hunk 28]
```diff
@@ -0,0 +1,161 @@
...
+
+Um die Integration zu erleichtern, stellen wir ein [OpenAPI Spezifikation](https://openapis.org) Dokument zur Verfügung, das die Endpunkte, Datenmodelle und Anfrage-/Antwortschemata der API beschreibt.
+Die API entspricht der Version 3.0.3 der OAS-Spezifikation.
+Du kannst unser aktuelles OAS document [hier](https://api.potterdb.com/v1/openapi.json) finden.
+
+## JSON:API Format
+
+Unsere REST-API entspricht der [JSON:API-Spezifikation](https://jsonapi.org/), die eine einheitliche Methode zur Strukturierung von API-Anfragen und -Antworten bietet. Hier ist ein Überblick über die wichtigsten Merkmale von JSON:API im Zusammenhang mit unserer API:
+
+- **Ressourcen Elemente**: Jede Ressource ist ein Objekt mit einem Typ (`type`), ID (`id`) und Attributen (`attributes`).
```
Comment (danielschuster-muc) [inline docs/pages/apis/rest.de.mdx:48]:
```suggestion
- **Ressourcen**: Jede Ressource ist ein Objekt mit einem Typ (`type`), ID (`id`) und Attributen (`attributes`).
```

File: docs/pages/apis/rest.de.mdx [hunk 29]
```diff
@@ -0,0 +1,161 @@
...
+- **Verbindungen**: Ressourcen können mit anderen Ressourcen verbunden sein, z. B. ist eine `Buch`-Ressource mit einer `Kapitel`-Ressource verbunden und umgekehrt.
+- **Paginierung**: Wenn eine Antwort eine große Anzahl von Ressourcen enthält, wird die Antwort paginiert. Siehe [Paginierung](#Paginierung) für weitere Infos.
```
Comment (danielschuster-muc) [inline docs/pages/apis/rest.de.mdx:50]:
```suggestion
- **Paginierung**: Wenn eine Antwort eine große Anzahl von Ressourcen enthält, wird die Antwort paginiert. Siehe [Paginierung](#paginierung) für weitere Infos.
```

File: docs/pages/apis/rest.de.mdx [hunk 30]
```diff
@@ -0,0 +1,161 @@
...
+- **Fehler**: Fehler werden mit standardisierten Strukturen zurückgegeben, was die Behandlung und Behebung von Problemen erleichtert.
```
Comment (danielschuster-muc) [inline docs/pages/apis/rest.de.mdx:51]:
```suggestion
- **Fehler**: Fehler werden in standardisiertem Format zurückgegeben, was die Behandlung und Behebung von Problemen erleichtert.
```

File: docs/pages/apis/rest.de.mdx [hunk 31]
```diff
@@ -0,0 +1,161 @@
...
+
+## Abfragen machen
+
+Um eine Abfrage an die REST-API zu machen, musst du eine "GET"-Anfrage an einen unserer verfügbaren Endpunkte stellen.
```
Comment (danielschuster-muc) [inline docs/pages/apis/rest.de.mdx:55]:
```suggestion
Um eine Abfrage an unsere REST-API zu machen, musst eine "GET"-Anfrage an einen unserer verfügbaren Endpunkte gestellt werden.
```

File: docs/pages/apis/rest.de.mdx [hunk 32]
```diff
@@ -0,0 +1,161 @@
...
+Hier ein einfaches Beispiel für das Abrufen einer Liste von Figuren über den Endpunkt "/characters":
```
Comment (danielschuster-muc) [inline docs/pages/apis/rest.de.mdx:56]:
```suggestion
Hier ein einfaches Beispiel für das Abrufen einer Liste von Figuren über den Endpunkt `/characters`:
```

File: docs/pages/apis/rest.de.mdx [hunk 33]
```diff
@@ -0,0 +1,161 @@
...
+
+### Sortieren
+
+Du kannst Ressourcen sortieren, indem du einen `sort` Abfrageparameter zu deiner Anfrage hinzufügst. Um beispielsweise Figuren nach Namen zu sortieren, kannst das Attribut `name` verwenden (unter [Ressourcen](#Ressourcen) findest du alle verfügbaren Attribute):
+
+```http
+GET https://api.potterdb.com/v1/characters?sort=name
+```
+
+Verwende das Präfix "-", um in absteigender Reihenfolge zu sortieren.
```
Comment (danielschuster-muc) [inline docs/pages/apis/rest.de.mdx:161]:
```suggestion
Verwende das Präfix `-`, um in absteigender Reihenfolge zu sortieren.
```

File: docs/pages/apis/rest.de.mdx [hunk 34]
```diff
@@ -0,0 +1,161 @@
...
+  - `'false'`: ist falsch 
+  - `present`: ist vorhanden 
+  - `blank`: ist leer
+  - `'null'`: ist null 
+  - `not_null`: ist nicht null
+</details>
+
+### Sortieren
+
+Du kannst Ressourcen sortieren, indem du einen `sort` Abfrageparameter zu deiner Anfrage hinzufügst. Um beispielsweise Figuren nach Namen zu sortieren, kannst das Attribut `name` verwenden (unter [Ressourcen](#Ressourcen) findest du alle verfügbaren Attribute):
```
Comment (danielschuster-muc) [inline docs/pages/apis/rest.de.mdx:155]:
```suggestion
Du kannst Ressourcen sortieren, indem du einen `sort` Abfrageparameter zu deiner Anfrage hinzufügst. Um beispielsweise Figuren nach Namen zu sortieren, kannst das Attribut `name` verwenden (unter [Ressourcen](/resourcen) findest du alle verfügbaren Attribute):
```
please test if it fixed it

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
Pretty solid, thank you! 
I am a bit picky regarding small things, so please resolve the requested changes. Makes it hard for me, as I am a native speaker :joy:-
Also regarding the "Kennung", I think Slug is understandable in the slang of programming, so I wouldn't translate it. 
Could you also try to add an empty line after each #TITLE? 
e.g.:
```
# Title
 
text starts here, after an empty line.
```

**Comment by kolja-lucht** [comment]
Thanks for your review! I had a feeling you'd be a little stricter with German. All good 😀 Will push the changes during the day 👍

**Comment by kolja-lucht** [comment]
>Could you also try to add an empty line after each #TITLE?

Do you mean only the top level titles or subheadings, too, like `## What's the Potter DB: API?` @danielschuster-muc

**Comment by kolja-lucht** [comment]
@danielschuster-muc anything I can do to fix the vercel deployment?

**Comment by danielschuster-muc** [comment]
Also regarding the failed build, here's an extract from the vercel log:
```
[nextra] Error compiling /vercel/path0/docs/pages/resources/spells.de.mdx.
--
15:37:05.109 | [nextra] The repository is shallow cloned, so the latest modified time will not be presented. Set the VERCEL_DEEP_CLONE=true environment variable to enable deep cloning.
15:37:22.964 | <w> [webpack.cache.PackFileCacheStrategy] Skipped not serializable cache item 'Compilation/modules\|/vercel/path0/docs/node_modules/next/dist/build/webpack/loaders/next-swc-loader.js??ruleSet[1].rules[12].use[0]!/vercel/path0/docs/node_modules/nextra/loader.js??ruleSet[1].rules[12].use[1]!/vercel/path0/docs/pages/resources/spells.de.mdx': No serializer registered for VFileMessage
15:37:22.964 | <w> while serializing webpack/lib/cache/PackFileCacheStrategy.PackContentItems -> webpack/lib/NormalModule -> webpack/lib/ModuleBuildError -> VFileMessage
15:37:23.686 | Failed to compile.
15:37:23.686 |  
15:37:23.686 | ./pages/resources/spells.de.mdx
15:37:23.686 | Could not parse expression with acorn: Unexpected token
15:37:23.686 |  
15:37:23.686 | Import trace for requested module:
15:37:23.686 | ./pages/resources/spells.de.mdx
15:37:23.686 |  
15:37:23.686 |  
15:37:23.686 | > Build failed because of webpack errors
15:37:23.801 | Error: Command "npm run build" exited with 1
```
looks like there is an issue with the spells resources page, please fix that.

File: docs/pages/apis/rest.de.mdx [hunk 35]
```diff
@@ -0,0 +1,161 @@
...
+  - `'false'`: ist falsch 
+  - `present`: ist vorhanden 
+  - `blank`: ist leer
+  - `'null'`: ist null 
+  - `not_null`: ist nicht null
+</details>
+
+### Sortieren
+
+Du kannst Ressourcen sortieren, indem du einen `sort` Abfrageparameter zu deiner Anfrage hinzufügst. Um beispielsweise Figuren nach Namen zu sortieren, kannst das Attribut `name` verwenden (unter [Ressourcen](/resourcen) findest du alle verfügbaren Attribute):
```
Comment (danielschuster-muc) [inline docs/pages/apis/rest.de.mdx:155]:
```suggestion
Du kannst Ressourcen sortieren, indem du einen `sort` Abfrageparameter zu deiner Anfrage hinzufügst. Um beispielsweise Figuren nach Namen zu sortieren, kannst das Attribut `name` verwenden (unter [Ressourcen](/resources) findest du alle verfügbaren Attribute):
```

File: docs/pages/resources/spells.de.mdx [hunk 36]
```diff
@@ -0,0 +1,89 @@
...
+      {
+        name: "name",
+        type: "Zeichenfolge",
+        description: "Name des Zauberspruchs.",
+        filterable: true,
+      },
+      {
+        name: "slug",
+        type: "Zeichenfolge",
+        description: Slug, um den Zauberspruch zu identifizieren.",
```
Comment (danielschuster-muc) [inline docs/pages/resources/spells.de.mdx]: @kolja-lucht the issue with the build probably lies here.

File: docs/pages/resources/characters.de.mdx [hunk 37]
```diff
@@ -0,0 +1,55 @@
...
+      <th>Typ</th>
+      <th>Beschreibung</th>
+      <th>Filterbar</th>
+    </tr>
+  </thead>
+  <tbody>
+    {[
+      { name: "alias_names", type: "Array", description: "Eine Liste von Namen, unter denen diese Figur sonst bekannt war.", filterable: true },
+      { name: "animagus", type: "Zeichenfolge", description: "Animagus der Figur.", filterable: true },
+      { name: "blood_status", type: "Zeichenfolge", description: "Blut-Status der Figur.", filterable: true },
```
Comment (danielschuster-muc) [inline docs/pages/resources/characters.de.mdx:21]:
```suggestion
      { name: "blood_status", type: "Zeichenfolge", description: "Blutstatus der Figur.", filterable: true },
```

File: docs/pages/resources/movies.de.mdx [hunk 38]
```diff
@@ -0,0 +1,137 @@
...
+      {
+        name: "screenwriters",
+        type: "Array",
+        description: "Liste der Personen, die das Drehbuch für den Film geschrieben haben.",
+        filterable: true,
+      },
+      {
+        name: "slug",
+        type: "Zeichenfolge",
+        description: "Slug um den Film zu identifizieren.",
```
Comment (danielschuster-muc) [inline docs/pages/resources/movies.de.mdx:100]:
```suggestion
        description: "Slug, um den Film zu identifizieren.",
```

File: docs/pages/resources/characters.de.mdx [hunk 39]
```diff
@@ -0,0 +1,55 @@
...
+      { name: "wiki", type: "Zeichenfolge", description: "Link zu der Harry Potter Wiki Seite der Figur.", filterable: false },
+    ].map((item) => (
+      <tr key={item.name}>
+        <td>{item.name}</td>
+        <td>{item.type}</td>
+        <td>{item.description}</td>
+        <td>{item.filterable == true ? '✅' : '❌'}</td>
+      </tr>
+    ))}
+
```
Comment (danielschuster-muc) [inline docs/pages/resources/characters.de.mdx:52]:
```suggestion

```

File: docs/pages/resources/movies.de.mdx [hunk 40]
```diff
@@ -0,0 +1,137 @@
...
+      {
+        name: "title",
+        type: "Zeichenfolge",
+        description: "Filmtitel.",
+        filterable: true,
+      },
+      {
+        name: "trailer",
+        type: "Zeichenfolge",
+        description: "Link zum Film-Trailer.",
```
Comment (danielschuster-muc) [inline docs/pages/resources/movies.de.mdx:118]:
```suggestion
        description: "Link zum Trailer des Films.",
```

File: docs/pages/index.de.mdx [hunk 41]
```diff
@@ -0,0 +1,95 @@
...
+Willkommen in der Dokumentation der Potter DB: API!
+Tauche ein in die magische Welt von Harry Potter mit Zugang zu Figuren, Büchern, Zaubersprüchen und mehr.
+Egal, ob Entwickler/in einer Potter-App oder nur ein neugieriger Fan, diese Dokumentation wird dich in die Geheimnisse unserer API einführen.
+
+## Was ist die Potter DB: API?
+
+Die Potter DB: API ist eine REST- und GraphQL-basierte Schnittstelle zu einem riesigen Datenbestand aus der zauberhaften Welt von Harry Potter. Wir kümmern uns um die Daten, damit du dich auf die Entwicklung deiner App konzentrieren kannst. Mit dieser API kannst du:
+- detaillierte Informationen über deine geliebten `Figuren` abrufen, einschließlich ihrer Hintergründe, Hauszugehörigkeit und mehr.
+- umfassende Listen von `Zaubersprüchen` und `Zaubertränken`, entdecken, die in der Welt der Zauberer verwendet werden.
+- ausführliche Informationen zu den `Büchern` und `Filmen` des Harry-Potter-Universums bekommen
```
Comment (danielschuster-muc) [inline docs/pages/index.de.mdx:87]:
```suggestion
- ausführliche Informationen zu den `Büchern` und `Filmen` des Harry-Potter-Universums bekommen.
```

File: docs/pages/resources/books.de.mdx [hunk 42]
```diff
@@ -0,0 +1,83 @@
...
+      {
+        name: "release_date",
+        type: "Datum",
+        description: "Erscheinungsdatum des Buchs.",
+        filterable: true,
+      },
+      {
+        name: "summary",
+        type: "Zeichenfolge",
+        description: "Buchzusammenfassung.",
```
Comment (danielschuster-muc) [inline docs/pages/resources/books.de.mdx:52]:
```suggestion
        description: "Zusammenfassung des Buches.",
```

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
Just some other small changes.

File: docs/pages/resources/spells.de.mdx (hunk 36 above)
Comment (kolja-lucht) [reply docs/pages/resources/spells.de.mdx]: Thanks, I already pushed the change. The quote in front of slug was missing. Build is fixed

File: docs/pages/apis/rest.de.mdx (hunk 35 above)
Comment (kolja-lucht) [reply docs/pages/apis/rest.de.mdx]: leads you to the correct page now

**Comment by kolja-lucht** [comment]
> Could you please resolve finished conversations? That makes it easier for me to track your changes, thanks!

@danielschuster-muc Build is fixed, I pushed your recent change suggestions and resolved all conversations

File: docs/pages/resources/movies.de.mdx [hunk 43]
```diff
@@ -0,0 +1,137 @@
...
+      {
+        name: "poster",
+        type: "Zeichenfolge",
+        description: "Link zum Poster des Films.",
+        filterable: false,
+      },
+      {
+        name: "producers",
+        type: "Array",
+        description: "Liste der Leute, die den Film produziert haben.",
```
Comment (danielschuster-muc) [inline docs/pages/resources/movies.de.mdx:70]:
```suggestion
        description: "Liste der Personen, die den Film produziert haben.",
```

File: docs/pages/resources/movies.de.mdx [hunk 44]
```diff
@@ -0,0 +1,137 @@
...
+      {
+        name: "editors",
+        type: "Array",
+        description: "Liste der Leute, die den Film geschnitten haben.",
+        filterable: true,
+      },
+      {
+        name: "music_composers",
+        type: "Array",
+        description: "Liste mit Komponisten der Filmmusik.",
```
Comment (danielschuster-muc) [inline docs/pages/resources/movies.de.mdx:58]:
```suggestion
        description: "Liste der Komponisten der Filmmusik.",
```

File: docs/pages/resources/movies.de.mdx [hunk 45]
```diff
@@ -0,0 +1,137 @@
...
+      {
+        name: "distributors",
+        type: "Array",
+        description: "Filmverleih.",
+        filterable: true,
+      },
+      {
+        name: "editors",
+        type: "Array",
+        description: "Liste der Leute, die den Film geschnitten haben.",
```
Comment (danielschuster-muc) [inline docs/pages/resources/movies.de.mdx:52]:
```suggestion
        description: "Liste der Personen, die den Film geschnitten haben.",
```

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
Almost

**Comment by kolja-lucht** [comment]
Pushed the changes

**Review by danielschuster-muc — APPROVED** [review]
Ein Träumchen 😹 

Thanks for you contribution!

**Pull request merged**